### DISCUSSION TITLE
Grouping of people under a name/ Creating album based on recognised faces.

### DISCUSSION THREAD

**Discussion by amgschnappi** [photoprism/photoprism#2209] [Ideas]
I have dumped all my google photos on my laptop and indexed the photos, tagged each person etc etc.

Now, I would like to have a grouping of photos as SchoolGrp, CollegeGrp, CoreFamilyGrp etc. as an Album/Group.

Then, in my CoreFamilyGrp, I will just add my parents, my spouse, my children names.
So all the photos of these people should be visible in this "Album".

Similarly, I will add Friend1, Friend2, Friend3, Friend4, Friend5.. etc in another Album/SchoolGrp. Then when I visit this Album/Group, all the photos of these people list should be shown.

I am not sure if such a thing exists. But would be great to have, if it doesnt exist and is developed for future releases.

**Comment by graciousgrey** [comment]
We a a few open issues that should cover this use case: https://github.com/photoprism/photoprism/issues/1553 https://github.com/photoprism/photoprism/issues/1431 https://github.com/photoprism/photoprism/issues/144
